platform/src/pages/opinions/new/page.rs

```diff
@@ -23,8 +24,9 @@ pub struct OpinionProps {
 
 #[component]
 pub fn OpinionCreatePage(props: OpinionProps) -> Element {
+    let popup_service: PopupService = use_context();
     let translates: OpinionNewTranslate = translate(&props.lang.clone());
-    let ctrl = Controller::new(props.lang, translates.clone());
+    let ctrl = Controller::new(props.lang, translates.clone(), popup_service);
```

Comment:
```suggestion
    let ctrl = Controller::new(props.lang);
```
이렇게만 넘기고, 나머지는 controller 안에서 use_context 와 translate 를 하시면 됩니다.

Reply:
수정했습니다!